Library/Homebrew/cmd/help.rb

```diff
@@ -15,7 +15,7 @@
 
 Developers:
   brew create [URL [--no-fetch]]
-  brew edit [FORMULA...]
+  brew (where|edit) [FORMULA...]
```

Comment:
sure, i did that because:
* `brew (info|home|options)` does it above
* i remember seeing somewhere to keep vertical size of --help small
* as i described in https://github.com/Homebrew/homebrew-core/issues/8935 `brew where` seems to supersede `brew edit` , `brew cat`
shall i go ahead w ur change?